blog/_posts/2018-11-18-RGSoCTeamsGOTOBerlin.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Some general comments about images:
Please remove the third image (`img/blog/2018/teamSectumsempra-GHCI2018.jpg`) in this pull request as it's not specific to this PR. This way we avoid mixing up things :) Also, the images are pretty big, so I'd ask you to resize/compress them. You can find some specific information about this, as well as links to services where you can compress images free of charge, [in our contribution guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). Please have them be at least smaller than 1MB. Thanks!